common/ethnicities/wc_ethnicities_troll.txt

```diff
@@ -0,0 +1,668 @@
+		1 = { name = mouth_lower_lip_pad_neg    range = { 0 1 }    }
+	}
+	gene_bs_mouth_lower_lip_width = {
+		1 = { name = mouth_lower_lip_width_pos    range = { 0 0.4 }    }
+	}
+	
+	###Nose
+	gene_bs_nose_length = {
+        1 = { name = nose_length_pos    range = { 0.4 1 }    }
+	}
+	gene_bs_nose_size = {
+        1 = { name = nose_size_pos    range = { 0.4 1 }    }
+	}
+	gene_bs_nose_forward = {
+        1 = { name = nose_forward_pos    range = { 0.6 1 }    }
+	}
+	gene_bs_nose_height = {
+        1 = { name = nose_height_pos    range = { 0 0 }    }
+		1 = { name = nose_height_neg   range = { 0 0.4 }    }
+	}
+	face_detail_nasolabial = {
+        1 = { name = nasolabial_01    range = { 0.5 1 }    }
+		1 = { name = nasolabial_02    range = { 0.5 1 }    }
+		1 = { name = nasolabial_03    range = { 0.5 1 }    }
+		1 = { name = nasolabial_04    range = { 0.5 1 }    }
+	}
+	
+	#Nose tip
+	gene_bs_nose_tip_forward = {
+		1 = { name = nose_tip_forward_pos    range = { 0.5 1 }    }
+	}
+	gene_bs_nose_tip_width = {
+		 1 = { name = nose_tip_width_neg    range = { 0.6 1 }    }
+	}
+	gene_bs_nose_tip_angle = {
+        1 = { name = nose_tip_angle_neg    range = { 0.6 1   }    }
+	}
+	face_detail_nose_tip_def = {
+        1 = {  name = nose_tip_def             range = { 0.5 1 }     }
+	}
+	
+	#Ridge
+	gene_bs_nose_profile = {
+        1 = { name = nose_profile_pos    range = { 0.6 1 }    }
+		1 = { name = nose_profile_hawk_pos    range = { 0.25 0.5 }    }
+	}
+	gene_bs_nose_ridge_angle = {
+        1 = { name = nose_ridge_angle_pos    range = { 0 1 }    }
+	}
+	gene_bs_nose_ridge_width = {
+        1 = { name = nose_ridge_width_pos    range = { 0 0.8 }    }
+		1 = { name = nose_ridge_width_neg    range = { 0 0.5 }    }
+	}
+	face_detail_nose_ridge_def = {
+        1 = { name = nose_ridge_def_pos    range = { 0 1 }    }
+		1 = { name = nose_ridge_def_neg    range = { 0 1 }    }
+	}
+	
+	#Nostril
+	gene_bs_nose_nostril_height = {
+        1 = { name = nose_nostril_height_pos    range = { 0.18 0.5   }    }
+	}
+	gene_bs_nose_nostril_width = {
+        1 = { name = nose_nostril_width_pos    range = { 0.3 0.65 }    }
+	}
+	
+	###Ear
+	gene_bs_ear_lenght = {
+        1 = { name = back_ear_lenght    range = { 1 1 } }
+    }
+	# gene_bs_ear_angle = {
+        # 1 = { name = ear_angle_pos    range = { 0.34 0.34 } }
+    # }
+	# gene_bs_ear_inner_shape = {
+        # 1 = { name = ear_inner_shape_pos    range = { 0.18 0.18 } }
+    # }
+	# gene_bs_ear_bend = {
+        # 1 = { name = ear_upper_bend_pos    range = { 0.05 0.05 } }
+    # }
+	# gene_bs_ear_outward = {
+        # 1 = { name = ear_outward_pos    range = { 0 0.5 } }
+		# 1 = { name = ear_outward_neg   range = { 0 1 } }
+    # }
+	# gene_bs_ear_size = {
+        # 1 = { name = ear_size_neg    range = { 0 1 } }
+		# 1 = { name = ear_size_pos    range = { 0 1 } }
+    # }
+	
+	###Forehead
+	gene_forehead_angle = {
+		1 = { name = forehead_angle_neg    range = { 0.35 0.5 }    }
+	}
```

Comment:
Don't you want to make it more negative? It visually makes their top head smaller and that's a very troll feature.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/107882892-4484b700-6f05-11eb-9dc4-d65ab28b2922.png)

</details>

Reply:
I suppose I can lower it a little bit, down to 0.15 or so. I wouldn't put it at 0 because at that point it starts to look like they're missing half their brains.

Reply:
Changed this in bdb09bc.